client/stories/tag.stories.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
```

Comment:
Note: rename this file to `Tag.stories.js` with a capital `T` to match the naming of the other files